jbake-core/src/main/java/org/jbake/app/Oven.java

```diff
@@ -216,7 +217,7 @@ private void resetDocumentTypesAndExtractors() {
     /**
      * Load {@link RenderingTool} instances and delegate rendering of documents to them
      */
-    private void renderContent() {
+    private void renderContent() throws InterruptedException {
```

Comment:
The method never checks if it got interrupted. The for loop would be a sane choice.